package.json

```diff
@@ -61,13 +53,10 @@
     "lint": "eslint --ext .js,.jsx src",
     "build": "npm run build:js && npm run build:scss",
     "build:js": "webpack -d",
-    "postbuild:js": "uglifyjs build/js/lyra.js -cm > build/js/lyra.min.js",
     "build:scss": "node-sass --include-path src/scss src/scss/main.scss build/css/main.css",
-    "postbuild:scss": "cleancss -o build/css/main.min.css build/css/main.css",
```

Comment:
we... need this one I think hehe

![screen shot 2016-03-02 at 1 33 17 pm](https://cloud.githubusercontent.com/assets/607541/13476179/6f12eb40-e07b-11e5-93c3-1ed237044012.png)